### ISSUE TITLE
User info bugs out upon creating portfolio

### ISSUE THREAD

**Issue by Chang-CH** [Colon-WQ/Portfolio#136]
### Steps to replicate

1. Login via Github
2. User/ profile picture should be displayed correctly
3. Create a portfolio
4. Add an entry (Might be optional)
5. Return to dashboard

### Result
![image](https://user-images.githubusercontent.com/37036869/124242871-cc372f00-db4f-11eb-94e9-cf1f73fbc6ad.png)
![image](https://user-images.githubusercontent.com/37036869/124243176-1ddfb980-db50-11eb-832b-df5cbf136fcf.png)

Issue seems similar to #114 

clearing site data and cookies does not solve the issue. Mongo was reset as well.

**Comment by twothicc** [comment]
I cannot replicate this issue. Tried clearing localStorage and cookies, followed by resetting mongodb table and following the same steps. The console log as I have checked out is as intended as well. You visited dashboard initially, created a portfolio and returned to dashboard without saving.

You visited dashboard twice, both times without having a user document saved in mongodb yet, hence two authentication procedures and two "user id not found" are printed in log.

I suspected that it could be a mistake with error handling of 404 error status returned for user id not found, which is not the case.

Chances are, your loggedIn boolean in localStorage is not updated to true correctly after you have logged in or after you returned to dashboard, leading to you being shown the dashboard meant for Guests.
